lib/stytch/otps.rb

```diff
@@ -1,209 +1,592 @@
 # frozen_string_literal: true
 
+# !!!
+# WARNING: This file is autogenerated
+# Only modify code within MANUAL() sections
+# or your changes may be overwritten later!
+# !!!
+
 require_relative 'request_helper'
 
 module Stytch
   class OTPs
     include Stytch::RequestHelper
-
     attr_reader :sms, :whatsapp, :email
 
-    PATH = '/v1/otps'
-
     def initialize(connection)
       @connection = connection
 
-      @sms = Stytch::OTPs::SMS.new(@connection)
-      @whatsapp = Stytch::OTPs::WhatsApp.new(@connection)
+      @sms = Stytch::OTPs::Sms.new(@connection)
+      @whatsapp = Stytch::OTPs::Whatsapp.new(@connection)
       @email = Stytch::OTPs::Email.new(@connection)
     end
 
+    # Authenticate a User given a `method_id` (the associated `email_id` or `phone_id`) and a `code`. This endpoint verifies that the code is valid, hasn't expired or been previously used, and any optional security settings such as IP match or user agent match are satisfied. A given `method_id` may only have a single active OTP code at any given time, if a User requests another OTP code before the first one has expired, the first one will be invalidated.
```

Comment:
so I'm not familiar with Ruby at all, but some googling tells me there are two common standards for generating docs. One is `yard`, the other `rdoc`. I managed to get both to run locally, and it looks like neither one of them support ``` `backtics` ``` for code formatting. Instead, they use either `+plusses+` or `<tt>teletype tags</tt>`

YARD:
![image](https://github.com/stytchauth/stytch-ruby/assets/89937743/72cc8530-f989-4e9c-80c7-c95f79e77dae)

RDOC:
![image](https://github.com/stytchauth/stytch-ruby/assets/89937743/e6817178-5fea-4468-83d7-a18cbb8f4256)

Reply:
Good to know. I can fix these.